client/state/inline-help/reducer.js

```diff
@@ -79,6 +79,7 @@ export const search = withoutPersistence(
 				return {
 					...state,
 					selectedResult: -1,
+					searchQuery: action.searchQuery,
```

Comment:
I think it will be enough to fix the issue. I think there isn't needed to add extra action.

Reply:
It's a good spot. I will work for the "My Home" version as we always dispatch the _action_ we just don't _track_ the search if there is no query (ie: it's empty):

https://github.com/Automattic/wp-calypso/blob/0f42e3ff06c2c40bc2d425c9bf3c9b9e07ea0b2a/client/my-sites/customer-home/cards/features/help-search/search-card.jsx#L91-L102

However, the FAB won't work as we conditionally dispatch the _action_ only when there is a search query:

https://github.com/Automattic/wp-calypso/blob/0f42e3ff06c2c40bc2d425c9bf3c9b9e07ea0b2a/client/blocks/inline-help/inline-help-search-card.jsx#L72-L86

So let's normalise the inline version so it matches the Home version and we'll be good.

This also shows how we need to extract a common set of components to avoid such issues.

I'll do this tomorrow AM.

Reply:
OK actually I've just fixed it.